client/src/components/LoginSession/LoginSession.js

```diff
@@ -119,19 +99,17 @@ function LoginSession(props) {
             , {lastActiveStr}
           </span>
         </p>
-        {props.IsCurrent &&
         <p>
-          <strong className={'text-success'}>
-            {currentStr}
-          </strong>
+          {props.IsCurrent &&
+            <strong className={'text-success'}>{currentStr}</strong>
+          }
+          {!props.IsCurrent && <a
+            role={'button'}
+            tabIndex={'0'}
```

Comment:
Why are we setting tab index to 0 here? That's preventing me from tabbing to the link.

Reply:
It's because it's an `<a>` tag without an href, so it needs a `role=button` + `tabindex=0` to make it tabbable.  It actually was tabbable before, it's just it lacked the `:focus` css and an `onKeyDown` handler. I've added all the things

Reply:
Updated